### PR TITLE
Add Arizona dashboard scraping for positive tests and cases

### PR DESCRIPTION
**Pull request by sarah-colq**

### CODE REVIEW

**Comment by sarah-colq** [comment]
Do not merge yet, we need to figure out how to acknowledge contributions.

**Comment by sarah-colq** [comment]
@igorkofman Please review.  Thanks!

File: src/cmdc_tools/datasets/official/AZ/data.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        ) 
+
+    def _get_testing(self):
+        viewPath = 'ELRcovid/LaboratoryTestingExternalDraft'
+        data = self._scrape_view(viewPath)
+        df = data['Map of Arizona']
+        renamed = df.rename(
+            columns={
+            "County (pt if avail, or provider))-value": "county",
+            "SUM(Number of Records)-alias": "positive_tests_total"
```
Comment (cc7768) [inline src/cmdc_tools/datasets/official/AZ/data.py]:
```suggestion
            "SUM(Number of Records)-alias": "tests_total"
```

File: src/cmdc_tools/datasets/official/AZ/data.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        })
+        renamed["county"] = renamed["county"].apply(lambda x: x.lower().capitalize())
+        renamed["dt"] = self._retrieve_dt("US/Mountain")
+        return (
+            renamed[["county", "dt", "positive_tests_total"]]
```
Comment (cc7768) [inline src/cmdc_tools/datasets/official/AZ/data.py]:
```suggestion
            renamed[["county", "dt", "tests_total"]]
```

**Review by cc7768 — APPROVED** [review]
@sarah-colq -- This is great work! They really worked to hide that data :anguished: 

The one change that I have put in this review is that it looks like you're extracting the total number of tests instead of the number of positive tests -- I added this change as a suggestion in the two places it should show up.

Let's wait to hear from Igor on best way to credit the SO question that you used, but otherwise this looks fantastic and is likely a spark to solving some other problems we've been waiting on.

**Comment by igorkofman** [comment]
Ship it (from an attribution standpoint)!

**Pull request merged**